### ISSUE TITLE
Unexpected unknown value `radial-gradient()` for property `background` (declaration-property-value-no-unknown)

### ISSUE THREAD

**Issue by xPaw** [stylelint/stylelint#8918]
### What minimal example or steps are needed to reproduce the bug?

```css
a {
	background: radial-gradient();
	background-image: radial-gradient();
}

b {
	background: radial-gradient(100% 100% at top left, hsl(291deg 63% 46%) 0%, hsl(207deg 90% 10%) 150%);
	background-image: radial-gradient(960px 2000px at top in oklch, #032962, #16224c, #191c37, #171724);
}
```

### What minimal configuration is needed to reproduce the bug?

see demo

### How did you run Stylelint?

https://stylelint.io/demo/#N4Igxg9gJgpiBcICGACYAdAdugLgIyTAGsBzAJwgFdMp4UykoBLJAGwFpzGmZMcAKAJQBuLLgLFyVGuyYBbJCRh0GzNp1U8+Q0ZgC+YzHjRj8hUhWq163dV2a8BARgAMLgKQpXHlEhwocCAAHFFYYADMcABoUAAsAZ1Z+ACYATidYEhQANgBmTwAWbPdBFA8YhKTklwB2TJRUn1cSrwBWDxFTCQtpKFkFJRVbDnstAVTslyCADxRqtxnff0CQpkwUCCJWMFiYgGIXXLTs5P2nE+SCsDP0sFyas5qnGsvO-RAokHCmMIA5JDkcEQMGmAKCYQAdGB4vEPuAIJhviQECAQUEIGR-LBwkhKKx-BhsDgQcSaPE6ABtADk8RwAE8wqw1jh2JBEUwSOxaUgaEgyFAqQBdKJYPS6OFspEAMQxChwKNpDJgTL4UIRSIhcgAVrDPrAgrDEISUCh0CBFYzmWa6GbWH4YLSzSL1qbzfTLXxWeqOVycDyoHyoNbXXbiY6QKKQHogA

### Which Stylelint-related dependencies are you using?

demo

### What did you expect to happen?

no error

### What actually happened?

It's reporting radial-gradient is an unknown property after latest stylelint update. (16.25.0 -> 16.26.1)

**Comment by ybiquitous** [comment]
@xPaw Thanks for the report. 

The [`declaration-property-value-no-unknown`](https://stylelint.io/user-guide/rules/declaration-property-value-no-unknown) heavily relies on the `@csstools/css-syntax-patches-for-csstree` package, which defines the syntax for the `radial-gradient()` function here:

```json
"radial-gradient-syntax": "[ [ [ <radial-shape> || <radial-size> ]? [ at <position> ]? ] || <color-interpolation-method> ]? , <color-stop-list>",
"radial-gradient()": "radial-gradient( [ <radial-gradient-syntax> ] )",
```

https://github.com/csstools/postcss-plugins/blob/b2e30d29760af3d54be90c2e8e24b6f9f7c5b3ed/packages/css-syntax-patches-for-csstree/dist/index.json#L760-L761

See also the rule implementation:
https://github.com/stylelint/stylelint/blob/39d8be97930679b7ced40a41bb7ce44caef55f53/lib/rules/declaration-property-value-no-unknown/index.mjs#L25

According to the [CSS spec](https://drafts.csswg.org/css-images/#radial-gradient-syntax) for the function, the package's syntax definition seems correct.

So, I'm wondering if your CSS code with `radial-gradient()` might be against the syntax definition, i.e., `radial-gradient( [ <radial-gradient-syntax> ] )`. Any thoughts? 🤔 

/cc @romainmenke If you have any info, please feel free to provide. 🙏🏼

**Comment by xPaw** [comment]
The empty ones was just to show the rule but I guess that's not correct, the other line with actual values works in browser but stylelint reports an error.

**Comment by ybiquitous** [comment]
Thanks for the quick confirmation. I'm not so familiar with the `radial-gradient()` function, but do you think the current `radial-gradient-syntax` definition in `@csstools/css-syntax-patches-for-csstree` would have any problems?

**Comment by ybiquitous** [comment]
Perhaps, you might be able to work around the issue by specifying a correct syntax to the [`propertiesSyntax`](https://stylelint.io/user-guide/rules/declaration-property-value-no-unknown/#propertiessyntax) secondary option of the rule. If you find anything there, we could apply any fix to `@csstools/css-syntax-patches-for-csstree` upstream.

**Comment by romainmenke** [comment]
Hi @xPaw,

Thank you for reporting this (and thank you @ybiquitous for pinging me).
This has been [fixed upstream](https://github.com/csstools/postcss-plugins/commit/82308a8a8b2759085360f10b4420e3cd20c4870a):

[Demo](https://stylelint.io/demo/#N4Igxg9gJgpiBcICGACYAdAdugLgIyTAGsBzAJwgFdMp4UykoBLJAGwFpzGmZMcAKAIwAGYQFIUI8SiQ4UOCAAcUrGADMcAGhQALAM6t+AJgCcg2CRQA2AMwSALFbEBKFOO37DR4QHYLKE2kRF0kAVnFnAG4sXAJicioadiYAWyQSGDoGZjZObJ4+fhMrYUUADxRvUXKZOQVlJkwUCCJWMB1tAGJhG1MrIy7BfqN7MEGzMBsfQZ9BHxGorABfEE0QNSZVADkkFLhEGDLdxVUAOjA9PVXwCEwNkgQQQ8UIMjlYNSRKVjkMbBxDgCaHo6ABtADkehwAE9VKxGjh2JA7kwSOwoUgaEgyFBwQBdTTLaKYa7I+4AMVeaRwjyhsJg8L451u91OKQAVlc1rBFFdEH8UCh0CA6XCEcK6MLWLIYFDhYSmkKRTCxXwkSzUeicJioNioBKldKAXKQMsQEsgA)

Updating your (indirect) dependencies should resolve this issue
